pyPlots/plot_colormap.py

```diff
@@ -231,50 +231,60 @@ def exprMA_cust(exprmaps, requestvariables=False):
     watermarkimageblack=os.path.join(os.path.dirname(__file__), 'logo_black.png')
     # watermarkimage=os.path.expandvars('$HOME/appl_taito/analysator/pyPlot/logo_color.png')
 
+    # Change certain falsy values:
+    if not lin and lin is not 0:
+        lin = None
+    if not symlog and symlog is not 0:
+        symlog = None
+    if symlog is True:
+        symlog = 0
+    if (filedir is ''):
```

Comment:
The checks here are explicitly to retain default behavior. Before if an empty string was passed it would not count as `None` (`'' != None` is true) and thus would be counted as the work directory. Now, the None checks are changed to truthiness checks so any empty strings are first changed explicitly to the work directory. This doesn't affect default values as `None is ''` is false.